### PR TITLE
SONiC Yang model for DHCP-Relay parameters

### PR DESCRIPTION
**Pull request by AkhileshSamineni**
SONiC Yang model for DHCP-Relay parameters

- What I did
Added DHCPv6 Servers.
Tables: VLAN, DHCP_RELAY.

- How I did it
Defined Yang model for Relay based on Guideline doc:
https://github.com/Azure/SONiC/blob/master/doc/mgmt/SONiC_YANG_Model_Guidelines.md

- How to verify it
Added test cases to verify it.

Signed-off-by: Akhilesh Samineni <akhilesh.samineni@broadcom.com>

### CODE REVIEW

**Review by shlomibitton — DISMISSED** [review]
LGTM

File: src/sonic-yang-models/tests/files/sample_config_db.json [hunk 1]
```diff
@@ -229,6 +232,9 @@
                 "dhcp_servers": [
                     "10.222.72.116"
                 ],
+                "dhcpv6_servers": [
```
[the remark this answers is not in the document]
Comment (AkhileshSamineni) [reply src/sonic-yang-models/tests/files/sample_config_db.json:235]: Added a new 'dhcpv6-relay' yang file changes for DHCP_RELAY table.

@kellyyeh Seeing parsing issues with leaf "dhcpv6_option|rfc6939_support", as it has '|' character.

[the remark this answers is not in the document]
Comment (kellyyeh) [reply src/sonic-yang-models/tests/files/sample_config_db.json:235]: Can we add it as just "rfc6939_support" please?

[the remark this answers is not in the document]
Comment (AkhileshSamineni) [reply src/sonic-yang-models/tests/files/sample_config_db.json:235]: Yes, added leaf as 'rfc6939_support'.

**Review by kellyyeh — APPROVED** [review]

**Comment by AkhileshSamineni** [comment]
@lguohan Please approve and merge it.

**Pull request merged**